### PR TITLE
Implement new design for Crawsqueal's main page

### PR DESCRIPTION
**Pull request by brandon-rusk**
Overhauled the design of the main page. This was a first pass and there is some broken functionality that still needs to be addressed. For the sake of this PR, the focus is on the style/design portion.

EDIT: Full Text selection now works in this PR.

Before:

image

After:

image

### CODE REVIEW

File: viewer/viewer/static_src/crawsqueal.css [hunk 1]
```diff
@@ -1,25 +1,33 @@
+.content_main-alt {
+  border-top: 0 !important;
+  width: 100% !important;
+}
+
 .search_results .results_count {
   padding-left: 1.875em;
+  padding-bottom: 15px;
+  margin-top: 30px;
+  height: 70px;
```
Comment (chosak) [inline viewer/viewer/static_src/crawsqueal.css:10]: This height causes a problem if there are no results:

image

File: viewer/viewer/templates/viewer/search_form.html [hunk 2]
```diff
@@ -1,20 +1,56 @@
...
-          </label>
-          <input id="query" name="q" type="text" title="Search terms" class="a-text-input" value="{{ request.query_params.q.strip }}" placeholder="Search terms">
-        </div>
-      </div>
-      <div class="o-form__input-w-btn_btn-container">
-        <button class="a-btn" aria-label="Search consumerfinance.gov content and source code">Search</button>
-      </div>
-    </div>
-  </form>
+	<form action="{% url 'index' %}" id="search_form">
```
Comment (chosak) [inline viewer/viewer/templates/viewer/search_form.html:2]: This indentation! 😆 I think we need a frontend linter on this project.

File: viewer/viewer/templates/viewer/search_side_bar.html [hunk 3]
```diff
@@ -1,4 +1,4 @@
-
+<!--
```
Comment (chosak) [inline viewer/viewer/templates/viewer/search_side_bar.html:1]: Better to delete this than comment out the whole file.

**Review by chosak — APPROVED** [review]
This is great! Some minor things to follow up on but I am going to merge and deploy this in the spirit of incremental progress. 🥳

File: viewer/viewer/templates/viewer/search_side_bar.html (hunk 3 above)
Comment (brandon-rusk) [reply viewer/viewer/templates/viewer/search_side_bar.html:1]: I plan to in the near future, however there is a small functionality I am still referencing. It should be removed by the next major PR

**Pull request merged**